examples/stackoverflow/README.md

```diff
@@ -1,3 +1,10 @@
 A StackOverflow client implemented using Riverpod and the official StackOverflow API
 
 The example is still in progress
+
+# Setup
+
+```
```

Comment:
```suggestion
This example uses code generation.
Before starting the application, you must first start the code generators.

This can be done with:

```
```